diagnostics-extension/src/dpsl.types.ts

```diff
@@ -236,3 +236,32 @@ export interface DiskReadRoutineParams {
   lengthSeconds: number;
   fileSizeMB: number;
 }
+
+export type DpslTypes =
+  | BacklightInfo
+  | BacklightInfoObject
+  | BacklightInfo
```

Comment:
added twice, also check other types (CpuInfo ... etc)

Reply:
and you don't need inner objects:

example: you need CpuInfo but not inner objects like: CpuCStateInfo, LogicalCpuInfo, PhysicalCpuInfo.

Same for the other types.